lib/cryptoexchange/exchanges/bitconnect/services/pairs.rb

```diff
@@ -0,0 +1,26 @@
+module Cryptoexchange::Exchanges
+  module Bitconnect
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Bitconnect::Market::API_URL}/info"
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          markets = output['markets']
+          markets.map do |value|
+            base, target = value['marketname'].split('_')
```

Comment:
I missed this out earlier, but this requires the same fix too @rusbal

Reply:
target/base correction